src/inference/include/openvino/runtime/intel_cpu/properties.hpp

```diff
@@ -26,6 +26,60 @@ namespace ov {
  */
 namespace intel_cpu {
 
+/**
+ * @enum       TbbPartitioner
+ * @brief      This enum contains definition of the type of TBB partitioner.
+ */
+enum class TbbPartitioner {
+    STATIC = 0,   //!<  Static partitioner
+    AUTO = 1,     //!<  Auto partitioner
+    DEFAULT = 2,  //!<  Initial value of configuration
```

Comment:
Does it actually mean the 'tbb_partitioner_auto'?
Let's make it more explicit.

Reply:
Since we have renamed it to "ADAPTIVE", it should be ADAPTIVE instead of a default.
And just a minor thing, is it probably a bit clearer to have a default parameter as a first one.
ADAPTIVE // select (for whole model, not per layer) either STATIC or AUTO based on heuristics
STATIC // force static partitioner
AUTO // force auto partitioner